### PR TITLE
Add multiObjectDelete action

### PR DESCRIPTION
**Pull request by LaurenSpiegel**
This should be merged with https://github.com/scality/Integration/pull/284

### CODE REVIEW

**Comment by michael-zapata** [comment]
Question: Shouldn't the metric count as multiple delete instead? Given that you can erase between 1 and 1000 objects in one go, we lose a lot of information only incrementing once.

File: src/lib/UtapiClient.js [hunk 1]
```diff
@@ -453,33 +453,38 @@ export default class UtapiClient {
...
     * @return {undefined}
     */
-    pushMetricDeleteObject(reqUid, bucket, objectSize, cb) {
+    _genericPushMetricDeleteObject(reqUid, bucket, objectSize,
+        numOfObjects, cb) {
         const callback = cb || this._noop;
         if (this.disableClient) {
             return callback();
         }
+        const bucketAction = numOfObjects === 1 ? 'deleteObject'
```
Comment (michael-zapata) [inline src/lib/UtapiClient.js]: Why not use a boolean instead to differ between the two operations?

**Comment by LaurenSpiegel** [comment]
@michael-zapata we're implementing a different api call.  We don't lose the info because we decrement the number of objects and size.

File: src/lib/UtapiClient.js (hunk 1 above)
Comment (michael-zapata) [inline src/lib/UtapiClient.js]: Ok, I did not see that these strings have purpose afterwards.

**Pull request merged**